api/src/main/java/org/openmrs/Concept.java

```diff
@@ -886,7 +886,7 @@ public ConceptName getShortNameInLocale(Locale locale) {
 			}
 		} else {
 			shortNames = getNames().stream()
-							.filter(n -> n.isShort())
+							.filter(ConceptName::isShort)
```

Comment:
How does this change make the code better?

Reply:
Method references are believed to be more readable, because they remove redundant "n ->" :)